doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -82,175 +76,107 @@ To work with the nRF54H20 DK, follow the instructions in the next sections to in
-      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+         #. Select :guilabel:`Open command prompt`.
 
-We recommend adding the nRF Util path to your environmental variables.
+         You can then follow the instructions in :ref:`creating_cmd`.
 
-.. _ug_nrf54h20_install_ncs:
-
-Installing the |NCS|
-====================
-
-After you have installed the toolchain, complete the following steps to get the customer sampling version of the |NCS|:
-
-1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling:
+Installing the Terminal application
+***********************************
 
-   .. parsed-literal::
-      :class: highlight
+On your computer, install `nRF Connect for Desktop`_.
+You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
 
-      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+Installing nRF Util and its commands
+************************************
 
-#. Enter the following command to clone the project repositories::
+Using the nRF54H20 DK with the |NCS| v2.7.0 requires the following:
 
-      west update
+* nRF Util version 7.11.1 or above
+* nRF Util ``device`` version 2.4.0
 
-   Depending on your connection, this might take some time.
+1. Download the nrfutil executable file from the `nRF Util development tool`_ product page.
+#. Add nRF Util to the system path (on Linux and MacOS) or environment variables (on Windows) to run it from anywhere on the system.
+   On Linux and MacOS, use one of the following options:
 
-#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
-   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+   * Add nRF Util's directory to the system path.
+   * Move the file to a directory in the system path.
 
-      west zephyr-export
+#. On MacOS and Linux, give ``nrfutil`` execute permissions by typing ``chmod +x nrfutil`` in a terminal or using a file browser.
+   This is typically a checkbox found under file properties.
+#. On MacOS, To run the nrfutil executable you need to allow it in the system settings.
+#. Verify the version of the nRF Util installation on your machine by running the following command::
 
-Your directory structure now looks similar to this::
+      nrfutil --version
 
-    ncs-lcs/work-dir
-    |___ .west
-    |___ bootloader
-    |___ modules
-    |___ nrf
-    |___ nrfxlib
-    |___ zephyr
-    |___ ...
+#. If your version is below 7.11.1, run the following command to update nRF Util::
 
-Note that there are additional folders, and that the structure might change.
-The full set of repositories and folders is defined in the manifest file.
+      nrfutil self-upgrade
```

Comment:
Please link here instead: https://nordic-dev.zoominsoftware.io/bundle/nrfutil/page/guides/installing.html

What is the point of me moving the nRF Util docs and updating them on weekly basis if we just duplicate info over and over?

Reply:
The getting started guide is supposed to be a one-stop shop for getting started with lilium. This section was tested during the last UX testing. I would not edit this for this release. We can unify this after 2.7